fuzzingandroid/FuzzerEngine/fuzzerengine/executor.py

```diff
@@ -364,7 +399,7 @@ def output(self):
 
     def dump_crash_logs(self):
         
-        cmd = "adb -s " + vm.VM.ip+ ":"+vm.VM.adb_port +"  logcat AndroidRuntime:E *:S"
+        cmd = "adb -s " + vm.VM.ip+ ":"+vm.VM.adb_port +"  logcat AndroidRuntime:E CrashAnrDetector:D System.err:W CustomActivityOnCrash:E ACRA:E WordPress-EDITOR:E *:F *:S"
```

Comment:
@tingsu @Zhendong2050: Register more oracle/crash related events? E.g. anything that prints to System.err is now considered as bug?